olive/systems/azureml/aml_system.py

```diff
@@ -92,6 +97,27 @@ def _create_environment(self, docker_config: AzureMLDockerConfig):
             return Environment(image=docker_config.base_image, conda_file=docker_config.conda_file_path)
         raise ValueError("Please specify DockerConfig.")
 
+    def _create_compute(self, aml_compute_config: AmlComputeConfig):
```

Comment:
what's the consideration that we need create aml compute? IMHO, the compute creation should be user's responsibility?

Reply:
@xiaoyu-work , should the aml compute creation happens when running python -m olive.workflows.run --setup stage?

Reply:
I would like we create the compute at runtime as this is a part of the process of olive workflow.